packages/core/src/common/uuid.ts

```diff
@@ -97,3 +99,13 @@ export const generateUuid = (function (): () => string {
         return result;
     };
 })();
+
+const NAMESPACE = '4c90ee4f-d952-44b1-83ca-f04121ab8e05';
+/**
+ * This function will hash the given value using SHA1. The result will be a uuid.
+ * @param value the string to hash
+ * @returns a uuid
+ */
+export function hashValue(value: string): string {
+    return v5(value, NAMESPACE);
```

Comment:
Since we are using `v5` as opposed to `v4` here, I assume that we made this choice on purpose to always get the same UUID given the specified value, right? If so I think it would be great to make that even more explicit in the comment. Or is that explicit enough in your opinion, just asking from a very novice point of view.

Reply:
Thanks, I've added a comment